xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-api/src/main/java/org/xwiki/mentions/script/MentionsScriptService.java

```diff
@@ -62,4 +66,28 @@ public String getSelfMentionsColor()
     {
         return this.configuration.getSelfMentionsColor();
     }
+
+    /**
+     * @see MentionsEventExecutor#getQueueSize()
+     * @return the current size of the queue of elements (page, comments...) with mentions to analyze
+     * @since 12.6RC1
+     */
+    @Unstable
+    public long getQueueSize()
+    {
+        return this.eventExecutor.getQueueSize();
+    }
+
+    /**
+     * Clear the queue of elements with mentions to analyze.
+     *
+     * @see MentionsEventExecutor#clearQueue()
+     * @since 12.6RC1
+     *
+     */
+    @Unstable
+    public void clearQueue()
```

Comment:
Sounds like something that would request some specific rights to trigger, maybe Admin rights at least?